### PR TITLE
 homework week2 wesam akfere 

### PR DESCRIPTION
**Pull request by wesam-k**

### CODE REVIEW

**Comment by wesam-k** [comment]
week2-homework-wesam-alkfere

File: Week2/homework/js-exercises/5-Who wants a drink.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    if (i < 4) {
+      drinkTray.push(drinkTypes[1]); 
+      continue;
+    }
+    if (i < 5) {
+      drinkTray.push(drinkTypes[2]); 
+    }
+  }
+ 
+  console.log('\"Hey guys, I brought a '+ drinkTray + '!\"');
```
Comment (marcianoviereck) [inline Week2/homework/js-exercises/5-Who wants a drink.js:22]: hah nice!

File: Week2/homework/js-exercises/Grade percentage (function).js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+'use strict'
+
+  
+
+
+function percentage(score){
+  
+  
+ if(score >= 90 && score <= 100){ //if the score between 90% to 100% the return a value is A
```
Comment (marcianoviereck) [inline Week2/homework/js-exercises/Grade percentage (function).js:10]: Can you add an extra `tab` here, for the identation. Then it's easier to read:)

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
The homework is looking good, I have some small remarks and I think you forgot the "calculator" project: "PROJECT: Grade calculator". Please check the assignment

**Comment by wesam-k** [comment]
done

**Pull request closed without merging**